services/storage-users/pkg/command/uploads.go

```diff
@@ -92,26 +90,31 @@ func PurgeExpiredUploads(cfg *config.Config) *cli.Command {
 				return err
 			}
 
-			managingFS, ok := fs.(storage.UploadsManager)
+			managingFS, ok := fs.(storage.UploadSessionLister)
 			if !ok {
 				fmt.Fprintf(os.Stderr, "'%s' storage does not support clean expired uploads\n", cfg.Driver)
 				os.Exit(1)
 			}
 
 			wg := sync.WaitGroup{}
 			wg.Add(1)
-			purgedChannel := make(chan tusd.FileInfo)
+			falseValue := false
+			trueValue := false
```

Comment:
This variable is very badly named. Is this intended?

Reply:
Hm, if I'm not mistaken you spotted a real issue there, not just a confusing name. I think we should briefly discuss that before merging

Reply:
I used the suffix `Value` because the UploadSessionFilter uses pointers for `Expired` and `Processing` to indicate if a filter is set or not.

we could use `expired := false`?

Reply:
and ... I'p prefer to not diverge from the naming in master ...

Reply:
> I used the suffix Value because the UploadSessionFilter uses pointers for Expired and Processing to indicate if a filter is set or not. 

I was not about the suffix. It is the prefix (`true`) that makes it badly named.

Reply:
fix in https://github.com/owncloud/ocis/pull/7958/files